embedded-graphics/src/primitives/rectangle.rs

```diff
@@ -60,20 +61,35 @@ impl Dimensions for Rectangle {
     }
 
     fn bottom_right(&self) -> Point {
-        self.bottom_right
+        if self.size == Size::zero() {
+            // FIXME: The bottom right corner isn't valid if the size of the rectangle is zero,
+            //        because creating a rectangle from two corner points creates a rectangle that
+            //        is at least 1x1 pixels large.
```

Comment:
Fixed by #182 as well?

Reply:
This is one of the problems mentioned in #182 that needs to be fixed later.